src/components/Avatar/Avatar.scss

```diff
@@ -73,7 +67,7 @@ $avatar-size-xs: 16px;
     }
   }
 
-  .remove-border {
+  .without-border {
```

Comment:
after change to using bem conventions it's will be: monday-style-avatar_circle--without-border